tests/e2e/test_load.py

```diff
@@ -0,0 +1,51 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See LICENSE in the project root for
+# license information.
+# --------------------------------------------------------------------------
+import pytest
+import torch
+from azstoragetorch.io import BlobIO
+
+
+@pytest.fixture(scope="module", autouse=True)
+def torch_hub_cache(tmp_path_factory):
+    current_dir = torch.hub.get_dir()
+    torch.hub.set_dir(tmp_path_factory.mktemp("torch_hub"))
+    yield
+    torch.hub.set_dir(current_dir)
+
+
+@pytest.fixture(scope="module")
+def model():
+    model = torch.hub.load("pytorch/vision:v0.10.0", "resnet101", pretrained=False)
+    return model
+
+
+@pytest.fixture(scope="module")
+def tmp_path(tmp_path_factory):
+    return tmp_path_factory.mktemp("model")
```

Comment:
We should be able to remove this fixture completely and just use the built-in `tmp_path`. According to `py.test` the built-in [`tmp_path`](https://docs.pytest.org/en/6.2.x/reference.html#std-fixture-tmp_path) will create a unique temporary directory for each invocation.